docs/infrastructure/installation/install-clio-on-ubuntu.md

```diff
@@ -10,7 +10,10 @@ labels:
 
 This page describes the recommended instructions for installing the latest stable version of Clio on **Ubuntu Linux 20.04 or higher** using the [`apt`](https://ubuntu.com/server/docs) utility.
 
-These instructions install a binary that has been compiled by Ripple. For instructions on how to build Clio from source, see the [Clio source code repository](https://github.com/XRPLF/clio).
+These instructions install an Ubuntu package that has been compiled and published by Ripple. You can also:
+
+- Download binaries, including for nightly and preview builds, from the [Clio releases page on GitHub](https://github.com/XRPLF/clio/releases/). (Expand the **Assets** section and choose the appropriate version for your OS.)
+- [Build Clio from source](https://github.com/XRPLF/clio/blob/develop/docs/build-clio.md).
```

Comment:
Maybe worth mentioning that we now also produce docker images: https://hub.docker.com/r/rippleci/clio